docker-compose/docker-compose.yml

```diff
@@ -28,40 +25,17 @@ services:
         condition: service_healthy
     environment:
       - MAX_HEAP_SIZE=1536M
-      - SEEDS=dse-1
-      - CLUSTER_NAME=dse-${DSETAG}-cluster
-      - DC=dc1
-      - RACK=rack1
-      - DS_LICENSE=accept
-    healthcheck:
-      test: [ "CMD", "cqlsh", "-u cassandra", "-p cassandra" ,"-e describe keyspaces" ]
-      interval: 15s
-      timeout: 10s
-      retries: 10
-
-  dse-3:
```

Comment:
Removed 3rd node to make similar to set up for core Stargate project; uses less resources too.